databases/week3/homework3.sql

```diff
@@ -0,0 +1,135 @@
+create Table Meal(
+ id int unsigned primary key auto_increment,
+ title varchar(255),
+ description text,
+ location varchar(255),
+ when_ date,
+ max_reservations int,
+ price decimal,
+ created_date date
+);
+
+
+create Table Reservation(
```

Comment:
The reservations table needs a column which show the date for the reservations

Reply:
i created the columns that it is already in the diagram